packages/Algernon.package/ALGItem.class/instance/matchesComment..st

```diff
@@ -3,4 +3,5 @@ matchesComment: aString
 
 	self algerType hasComment
```

Comment:
same here. I'm not sure we need this anymore.

Reply:
We don't; I removed it.